src/counters.c

```diff
@@ -817,6 +826,10 @@ static int StatsOutput(ThreadVars *tv)
                     table[x].value = (uint64_t)(m->value / m->updates);
                 }
                 break;
+            case STATS_TYPE_TIMED:
+                table[x].value = table[x].counter_value - table[x].counter_pvalue;
```

Comment:
I think if you work with the global table here (that aggregates results from individual workers) then for the pps/bps feature you should only add up these numbers together.

Say you have 4 worker threads and each worker experiences a load of e.g. 10 pps then the global throughput is 40 pps and not 10 pps.

Reply:
The counter_value and counter_pvalue have the total of pkts an bytes respectively to this Stats window and last one. They are used to calculate pps and bps directly in global table :sob: 

I tried calculating pps/bps on each worker thread, but I couldn't do it with a successful approach: under:. If you have any suggestions on how to do this, I would be very grateful.

So I use the fact that the global table will always give me the aggregated values, so I don't have to worry about a way to calculate pps/bps on each worker thread.